src/main/java/DataManipulation.java

```diff
@@ -0,0 +1,81 @@
+import static java.util.Arrays.asList;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import org.apache.arrow.memory.RootAllocator;
+import org.apache.arrow.vector.FieldVector;
+import org.apache.arrow.vector.VarCharVector;
+import org.apache.arrow.vector.VectorSchemaRoot;
+
+public class DataManipulation {
+  public static final RootAllocator allocator = new RootAllocator(Long.MAX_VALUE);
+
+  public List readCSVFile(String path) throws FileNotFoundException {
+    String line;
+    final VarCharVector nameVector = new VarCharVector("name", allocator);
+    final VarCharVector teamVector = new VarCharVector("team", allocator);
+    final VarCharVector positionVector = new VarCharVector("position",allocator);
+    final VarCharVector heightVector = new VarCharVector("height", allocator);
+    final VarCharVector weightVector = new VarCharVector("weight", allocator);
+    final VarCharVector  ageVector = new VarCharVector("age", allocator);
+    try (BufferedReader br = new BufferedReader(new FileReader(path))) {
+      int i = 0;
```

Comment:
Can we make this a attribute from the class? When you instantiate the class, it starts as 0. Then when you iterate you increment it. This way, the class will have the total of lines that the csv has and you can pass it to setRowCountValue instead of harded coding

Reply:
Since you've create the rowCount variable you don't need the i variable anymore, Use the rowCount